index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/4e96f49cc5.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <a href="#"><h1 class="nav-branding">My Logo</h1></a>
+        <ul class="nav-menu">
+          <li class="nav-item nav-link"><a href="#">Portfolio</a></li>
+          <li class="nav-item nav-link"><a href="#">About</a></li>
+          <li class="nav-item nav-link"><a href="#">Contact</a></li>
+        </ul>
+        <div class="hambuger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <!-- Headline -->
+
+    <section class="headline-bg headline">
+      <h1 class="headline-main">
+        I'm Promise <br />
+        Glad to see you!
+      </h1>
+      <p class="headline-paragraph">
+        I’m a software developer! I can help you build a product , feature or
+        website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3 class="headline-connect">LET'S CONNECT</h3>
+      <div class="headline-icons">
+        <a href="#" class="fa-brands fa-twitter fa-xl"></a>
+        <a href="#" class="fa-brands fa-linkedin-in fa-xl"></a>
+        <a href="#" class="fa-brands fa-github fa-xl"></a>
+        <a href="#" class="fa-solid fa-m fa-xl"></a>
+        <a href="#" class="fa-solid fa-hand-peace fa-xl"></a>
+      </div>
```

Comment:
- [ ] Good implementation of changes required but in this section, the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag), also as mentioned by the previous reviewer kindly use the icons images that you have already download for all the social media(Github, Angelist, LinkedIn, Twitter, Medium) :upside_down_face: :upside_down_face: